pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -273,9 +273,11 @@ func (r *DNSProvider) getHostedZoneID(fqdn string) (string, error) {
 
 func newTXTRecordSet(fqdn, value string, ttl int) *route53.ResourceRecordSet {
 	return &route53.ResourceRecordSet{
-		Name: aws.String(fqdn),
-		Type: aws.String(route53.RRTypeTxt),
-		TTL:  aws.Int64(int64(ttl)),
+		Name:             aws.String(fqdn),
+		Type:             aws.String(route53.RRTypeTxt),
+		TTL:              aws.Int64(int64(ttl)),
+		MultiValueAnswer: aws.Bool(true),
+		SetIdentifier:    aws.String(value),
```

Comment:
Just to confirm a few things here:

1) This call to the AWS API only expects the *new* record values to be provided in the `value` here right? Not a full list of all the values that need to be set/present.

2) When *deleting* the record, does the AWS API know to only delete the one single value rather than the whole set of values associated with that FQDN?

Reply:
1. Yes, in the case of MultiValue records, AWS also needs a SetIdentifier to be able to discern through the multiple records.
2. When deleting it will use the same object as when it created it, so it will only delete one record.